bin/db-initialization.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+import os
+
+from buildpg import V, asyncpg, render
+
+DB_NAME = os.environ["DATABASE_TO_CREATE"]
```

Comment:
Generally speaking, you should avoid reading env vars as part of module initialization. It makes it difficult if you want to override them after importing.